ExternalBenchmarks/README.md

```diff
@@ -0,0 +1,46 @@
+# ExternalBenchmarks
+
+Additional benchmarks that utilize https://github.com/ordo-one/package-benchmark.
+
+This code is explicitly in a subdirectory with a local reference to swift-noise
+in order avoid adding transitive dependencies.
+
+To run the benchmarks, invoke the following command:
+
+    swift package benchmark
+
+## Using Benchmarks
+
+- [Documentation for the package](https://swiftpackageindex.com/ordo-one/package-benchmark/documentation/benchmark/gettingstarted)
```

Comment:
nit: could we link to the [Swiftinit version](https://swiftinit.org/docs/package-benchmark/benchmark) of the the `package-benchmark` docs? that is where the docs for swift-noise itself live.

Reply:
sure, no problem - I'll switch them up